server/controllers/FollowController.js

```diff
@@ -60,6 +79,17 @@ class FollowController {
                 })
                   .then((unfollowed) => {
                     if (unfollowed) {
+                      type = 'Unfollow';
+                      addNewNotificationEvent(
+                        'unfollows',
+                        authorId,
+                        id,
+                        'Someone unfollowed you',
```

Comment:
This message: `Someone unfollowed you`
Can you make this message more specific. example: `Abiodun suleiman` unfollowed you